packages/components/src/dropdown/test/index.js

```diff
@@ -1,98 +1,68 @@
 /**
  * External dependencies
  */
-import { fireEvent, render } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 
 /**
  * Internal dependencies
  */
-import Dropdown from '../';
-
-function getButtonElement( container ) {
-	return container.querySelector( 'button' );
-}
-function getOpenCloseButton( container, selector ) {
-	return container.querySelector( selector );
-}
+import Dropdown from '..';
 
 describe( 'Dropdown', () => {
-	function expectPopoverVisible( container, value ) {
-		const popover = container.querySelector( '.components-popover' );
-		if ( value ) {
-			expect( popover ).toBeTruthy();
-		} else {
-			expect( popover ).toBeFalsy();
-		}
-	}
-
-	it( 'should toggle the dropdown properly', () => {
-		const expectButtonExpanded = ( container, expanded ) => {
-			expect( container.querySelectorAll( 'button' ) ).toHaveLength( 1 );
-			expect( getButtonElement( container ) ).toHaveAttribute(
-				'aria-expanded',
-				expanded.toString()
-			);
-		};
-
-		const {
-			container: { firstChild: dropdownContainer },
-		} = render(
+	it( 'should toggle the dropdown properly', async () => {
+		render(
 			<Dropdown
-				className="container"
-				contentClassName="content"
 				renderToggle={ ( { isOpen, onToggle } ) => (
 					<button aria-expanded={ isOpen } onClick={ onToggle }>
-						Toggleee
+						Toggle
 					</button>
 				) }
-				renderContent={ () => <span>test</span> }
+				renderContent={ () => 'Content' }
 			/>
 		);
 
-		expectButtonExpanded( dropdownContainer, false );
-		expectPopoverVisible( dropdownContainer, false );
+		const user = userEvent.setup( {
+			advanceTimers: jest.advanceTimersByTime,
+		} );
+		const button = screen.getByRole( 'button' );
```

Comment:
If we add `expanded: false` to the query, we won't need the extra `expect( button ).toHaveAttribute( 'aria-expanded', 'false' );` below. We could make a new query for the next assertion where we test for `[aria-expanded=true]`. No big deal, just an optional suggestion you could consider.

```suggestion
		const button = screen.getByRole( 'button', { expanded: false } );
```

Reply:
Smart. Updated in 2d62d9b4f4006b09200779ce7333f3bf18503e6b